src/public/modules/forms/base/controllers/submit-form.client.controller.js

```diff
@@ -27,6 +27,15 @@ function SubmitFormController(
   // The form attribute of the FormData object contains the form fields, logic etc
   vm.myform = FormData.form
 
+  // For SP / CP forms, also include the spcpSession details
+  // This allows the log out button to be correctly populated with the UID
+  // Also provides time to cookie expiry so that client can refresh page
+  if (['SP', 'CP'].includes(vm.myform.authType)) {
+    if (FormData.spcpSession && FormData.spcpSession.userName) {
```

Comment:
would it be cleaner to combine these conditionals instead of splitting them into two `if` statements?

Reply:
thanks! combined